Dockerfile

```diff
@@ -1,29 +1,53 @@
-FROM ruby:2.5.5 as builder
+##########################################################################
+## Dockerfile for SA@OSU 
+##########################################################################
+FROM ruby:2.5-alpine3.12 as bundler
 
 # Necessary for bundler to properly install some gems
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 
-# add nodejs and yarn dependencies for the frontend
-RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - && \
-  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+# Install bundler
 RUN gem install bundler
-RUN apt-get update -qq && apt-get upgrade -y && \
-  apt-get install -y build-essential libpq-dev mariadb-client nodejs libreoffice imagemagick unzip ghostscript vim yarn
-
-# install clamav for antivirus
-# fetch clamav local database
-# RUN apt-get install -y clamav-freshclam clamav-daemon libclamav-dev clamav-base
-# RUN mkdir -p /var/lib/clamav && \
-#   wget -O /var/lib/clamav/main.cvd http://database.clamav.net/main.cvd && \
-#   wget -O /var/lib/clamav/daily.cvd http://database.clamav.net/daily.cvd && \
-#   wget -O /var/lib/clamav/bytecode.cvd http://database.clamav.net/bytecode.cvd && \
-#   chown clamav:clamav /var/lib/clamav/*.cvd
 
+##########################################################################
+## Install dependencies
+##########################################################################
+FROM bundler as dependencies
+
+RUN apk --no-cache update && apk --no-cache upgrade && \
+  apk add --no-cache alpine-sdk nodejs imagemagick unzip ghostscript vim yarn \
+  git sqlite sqlite-dev mysql mysql-client mysql-dev libressl libressl-dev \
+  curl libc6-compat build-base tzdata zip autoconf automake libtool texinfo \
+  bash bash-completion java-common openjdk11-jre-headless graphicsmagick \
+  poppler ffmpeg tesseract-ocr openjpeg-dev openjpeg-tools openjpeg lcms2 \
```

Comment:
since this is `alpine` the package names and their contents will be a little different. I've been using this tool for figuring out which packages satisfy whatever dependencies. 

https://pkgs.alpinelinux.org/contents?file=&path=&name=poppler&branch=v3.12&repo=main&arch=x86_64

if there are packages from OD2 we don't need we should absolutely remove them!